src/CalculateUtil.js

```diff
@@ -0,0 +1,28 @@
+import BASIC_SEPARATOR from "./constants/basicSeparator";
+
+const CalculateUtil = {
+  parseInput(input) {
+    const separator = [...BASIC_SEPARATOR];
+    const re = /(\/\/).+(\\n)/;
+
+    if (input.match(re)) {
+      input
+        .substring(2, input.indexOf("\\"))
+        .split("")
+        .map((sep) => separator.push(sep));
```

Comment:
```suggestion
        .forEach((sep) => separator.push(sep));
```
저는 `map`은 기존 배열을 새로운 배열로 변환할 때 사용하는데 지예님은 어떤 상황에 사용하시나요??

저라면 지금 상황에서는 `forEach`를 사용하거나 스프레드 문법으로 배열을 합칠 것 같아요!
```js
const customSeparators = input.substring(2, input.indexOf("\\")).split("");
const separators = [...customSeparators, ... separator];
```

Reply:
이 경우 `forEach`를 사용하는게 좋아보입니다! `map`은 습관처럼 사용했는데 `map`메서드의 경우, 새로운 배열을 반환하는 메서드라서 메모리도 한번 더 차지하게 되는 것 같더라구요. (이번에 알게된 사실 흑흑) 이렇게 단순 반복으로 사용할 때는 `forEach`를 사용하는 것이 여러모로 좋아보입니다! 👍 

스프레드 문법으로 합치는것도 너무 좋네요! `push` 생각했었는데.. 왜그랬지 ㅎㅎ 감사합니당 ☺️